ccmbioinfo/lfs
# CARDS-2233: Optimize the performance of the count and pagination servlets

To test:
1. build with `mvn clean install -Pdocker`
2. in `compose-cluster/mssql` run `python3 ./generate_test_YE_sql.py -n 500 --basedate 2023-06-01 --time_spread_seconds 0 prems_sample.sql` (use 7 days ago as the date)
3. in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --composum --adminer --clarity`
4. in `compose-cluster` start cards with `docker-compose build && docker-compose up`
5. at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the generated `prems_sample.sql` file
6. open `http://localhost:8080/Subjects.importClarity?dayToQuery=-7` and wait for about 20 minutes
7. check that the patients have been imported correctly
8. check that the user dashboard, clinic dashboards, and the patients/questionnaires tables show data correctly, support changing the page size and going to a different page, filtering, and don't show the same item more than once
9. does it feel a bit speedier than before?
